app/Watermark/Commands/CallbackqueryCommand.php

```diff
@@ -7,13 +7,17 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
 namespace Longman\TelegramBot\Commands\SystemCommands;
+
 use Longman\TelegramBot\Commands\SystemCommand;
-use Longman\TelegramBot\Request;
+use Longman\TelegramBot\Commands\UserCommands\EditmessageCommand;
 use Longman\TelegramBot\Conversation;
-
+use Longman\TelegramBot\Entities\ServerResponse;
+use Longman\TelegramBot\Exception\TelegramException;
+use Longman\TelegramBot\Request;
 use Yii;
```

Comment:
Remove this too !

Reply:
You mean you'd prefer to have it inline in the code?

Reply:
No, i mean this line > use Yii;